pkg/agent/attestor/node/node.go

```diff
@@ -56,14 +58,29 @@ type Config struct {
 	SVIDCachePath     string
 	Log               logrus.FieldLogger
 	ServerAddress     string
+
+	// Use experimental API
+	ExperimentalAPIEnabled bool
 }
 
 type attestor struct {
 	c *Config
+
+	// Used for testing purposes.
+	createNewAgentClient  func(*grpc.ClientConn) agent.AgentClient
+	createNewBundleClient func(*grpc.ClientConn) bundle.BundleClient
```

Comment:
this will need tweaking with the grpc update that just got merged (to swap `*grpc.ClientConn` for `grpc.ClientConnInterface)